infrastructure/resource_group_level/main.bicep

```diff
@@ -123,7 +123,7 @@ var externalApiSettings = {
 }
 var externalApiSecureSettings = {
   CORS_ORIGIN: CORS_ORIGIN
-  APIM_TFS_URL: APIM_TFS_URL
+  APIM_TFS_URL: APIM_TFS_URL // TODO:FN-1086 different between dev & GH.
```

Comment:
Note that the GitHub secret has the value `<blah>/mdm/api/v1/` but the dev export has `<blah>/tfs/api/v1`.
I suspect the secret is wrong.